apps/prairielearn/elements/pl-drawing/pl-drawing.py

```diff
@@ -34,7 +34,7 @@ def union_drawing_items(e1, e2):
 
     newobj = []
     for item in obj1:
-        if not item["id"] in new_ids:
+        if item["id"] not in new_ids:
```

Comment:
This was the only non-automatic change (though I think it actually can be done automatically as well).

Reply:
Also had to move some comments in PHS: https://github.com/PrairieLearn/PrairieLearn/pull/9744/commits/208589c00caa5f7bdf4af87e19cecc8d4f592c3e